CMakeLists.txt

```diff
@@ -178,7 +84,6 @@ target_link_libraries(${CMAKE_WORKSPACE_NAME}_${CURRENT_PROJECT_NAME} INTERFACE
                       ${CMAKE_WORKSPACE_NAME}::hash
                       ${CMAKE_WORKSPACE_NAME}::multiprecision
                       ${CMAKE_WORKSPACE_NAME}::containers)
-                    #   marshalling::crypto3_zk)
```

Comment:
We probably have this dependency, but we choose not to list it here, that's why we had this comment. But it's ok to delete it.